src/timeline.ts

```diff
@@ -205,18 +205,18 @@ class Timeline implements IAppView {
       .domain([0, timeRange])
       .range([20, widthTimelineDiv-20]); // 20 = Spacing
 
-
     svgtimeline.selectAll('circle')
       .data(items)
       .enter()
       .append('circle')
       .attr('title', (d:any) => (d.time) ? d.time.format(AppConstants.DATE_FORMAT) : d.key)
       .attr('cy', 60)
-      .attr('cx', (d:any) => {
+      .attr('cx', (d:any, i) => {
         if(d.time) {
           return xScaleTime(moment(d.time).diff(moment(items[0].time),'days'));
         } else {
-          return 60;
+          return i * ((widthTimelineDiv-20)/7);
```

Comment:
Or since I saw that you use the expression `((widthTimelineDiv-20)/7)` multiple times you can also wrap it in a re-usable function.

```
function xyz(widthTimelineDiv) {
  const myConst = 20;
  const myConst2 = 7;
  return (widthTimelineDiv - myConst) / myConst2;
}
```

Please rename the function and constant properly.